protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/copy_data_types.rs

```diff
@@ -13,15 +38,6 @@ impl Fixed for bool {
     const SIZE: usize = 1;
 }
 
-// Boolean value. Encoded as an unsigned 1-bit integer,
-// True = 1, False = 0 with 7 additional padding bits in
-// the high positions.
-// x
-// Recipients MUST NOT interpret bits outside of the
-// least significant bit. Senders MAY set bits outside of
-// the least significant bit to any value without any
-// impact on meaning. This allows future use of other
-// bits as flag bits.
```

Comment:
why are we removing these comments?

I don't have full context, but they seem useful?